### PR TITLE
Disabling font scaling in profile when system font size is increased

### PR DESCRIPTION
**Pull request by murchant**
Updating the BpkText comment so the resizing does of text does not occur when the phones text is increased through accessibility settings. This is currently the behaviour in native screens and this behaviour should be present in react native screens, and therefore over al screens. 

+ [x] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-react-native/blob/master/CONTRIBUTING.md)

Remember to include the following changes:
+ [x] `UNRELEASED.md`
+ [x] `README.md`
+ [x] Tests
+ [x] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

_If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/master/CODE_REVIEW_GUIDELINES.md)_

### CODE REVIEW

File: packages/react-native-bpk-component-text/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-bpk-component-text",
-  "version": "4.2.1",
+  "version": "4.2.2",
```
Comment (tiagoengel) [inline packages/react-native-bpk-component-text/package.json]: Was this updated manually? If so, can you rever it? This is handled by the release script.

File: UNRELEASED.md [hunk 2]
```diff
@@ -4,6 +4,9 @@
 
 **Added:**
 
+- react-native-bpk-component-text:
```
Comment (tiagoengel) [inline UNRELEASED.md]: Can you please move this to a `**Fixed:**` block?

File: packages/react-native-bpk-component-text/package.json (hunk 1 above)
Comment (murchant) [reply packages/react-native-bpk-component-text/package.json]: No I updated it, ok sounds good

File: UNRELEASED.md (hunk 2 above)
Comment (murchant) [reply UNRELEASED.md]: Yes ! will do

File: UNRELEASED.md [hunk 3]
```diff
@@ -51,3 +51,6 @@ See [`CHANGELOG.md`](CHANGELOG.md) for real-world examples of good changelog ent
 
 - react-native-bpk-component-horcrux:
   - Fixed issue where `BpkHorcrux` would occasionally possess the living.
+
```
Comment (tiagoengel) [inline UNRELEASED.md]: Looks good now, but this should be at the top of the file, this part is just examples.

**Pull request merged**